projects/Apps/crosswords/package.json

```diff
@@ -22,7 +22,7 @@
         "webpack-cli": "^3.3.11"
     },
     "dependencies": {
-        "@guardian/react-crossword": "^0.2.0",
+        "@guardian/react-crossword": "^0.1.8",
```

Comment:
I think we want to use `"0.1.8"` instead of `"^0.1.8"` to ensure it doesn't use a later version

Reply:
good spot, I'm not doing a very good job of copying a 4 character change here! fixed now